<?xml version="1.0" encoding="UTF-8"?><testrun name="EmployeeTestCase" project="ShoppingCartBackEnd" tests="5" started="5" failures="1" errors="0" ignored="0">
  <testsuite name="com.niit.shoppingcart.test.EmployeeTestCase" time="6.321">
    <testcase name="getEmployee" classname="com.niit.shoppingcart.test.EmployeeTestCase" time="4.875"/>
    <testcase name="createEmployee" classname="com.niit.shoppingcart.test.EmployeeTestCase" time="0.413"/>
    <testcase name="deleteEmployee" classname="com.niit.shoppingcart.test.EmployeeTestCase" time="0.363"/>
    <testcase name="updateEmployee" classname="com.niit.shoppingcart.test.EmployeeTestCase" time="0.363">
      <failure>java.lang.AssertionError: expected:&lt;Employee [eid=1001, name=updated name 1001, salary=60000.0]&gt; but was:&lt;Employee [eid=1001, name=updated name 1001, salary=60000.0]&gt;&#13;
	at org.junit.Assert.fail(Assert.java:69)&#13;
	at org.junit.Assert.failNotEquals(Assert.java:329)&#13;
	at org.junit.Assert.assertEquals(Assert.java:97)&#13;
	at org.junit.Assert.assertEquals(Assert.java:108)&#13;
	at com.niit.shoppingcart.test.EmployeeTestCase.updateEmployee(EmployeeTestCase.java:115)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.junit.internal.runners.TestMethodRunner.executeMethodBody(TestMethodRunner.java:99)&#13;
	at org.junit.internal.runners.TestMethodRunner.runUnprotected(TestMethodRunner.java:81)&#13;
	at org.junit.internal.runners.BeforeAndAfterRunner.runProtected(BeforeAndAfterRunner.java:34)&#13;
	at org.junit.internal.runners.TestMethodRunner.runMethod(TestMethodRunner.java:75)&#13;
	at org.junit.internal.runners.TestMethodRunner.run(TestMethodRunner.java:45)&#13;
	at org.junit.internal.runners.TestClassMethodsRunner.invokeTestMethod(TestClassMethodsRunner.java:75)&#13;
	at org.junit.internal.runners.TestClassMethodsRunner.run(TestClassMethodsRunner.java:36)&#13;
	at org.junit.internal.runners.TestClassRunner$1.runUnprotected(TestClassRunner.java:42)&#13;
	at org.junit.internal.runners.BeforeAndAfterRunner.runProtected(BeforeAndAfterRunner.java:34)&#13;
	at org.junit.internal.runners.TestClassRunner.run(TestClassRunner.java:52)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="getEmployeeList" classname="com.niit.shoppingcart.test.EmployeeTestCase" time="0.307"/>
  </testsuite>
</testrun>
